src/utilities/__init__.py

```
@@ -0,0 +1,7 @@
+"""This module defines all utility functions and classes."""
+
+from .pcall import pcall
```

Ruff (0.12.2)

`.pcall.pcall` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)